front/app/containers/Admin/settings/registration/ToggleShowFollowPreferences/index.tsx

```diff
@@ -35,13 +29,22 @@ const ToggleShowFollowPreferences = ({ isEnabled, onChange }: Props) => {
           />
         </Box>
 
-        <Text fontSize="base" color="textSecondary" pl="1rem" my="0px">
-          {isEnabled ? (
-            <FormattedMessage {...messages.enabled} />
-          ) : (
-            <FormattedMessage {...messages.disabled} />
-          )}
-        </Text>
+        <Box display="flex" flexDirection="column">
+          <Title
+            color="primary"
+            fontSize="l"
+            pl="1rem"
+            my="0px"
+            style={{ fontWeight: 500 }}
```

Comment:
An unrelated note - we should probably look into adding some more font weights into our component library :thinking: I often need to use semibold (600) for stuff I'm implementing as well. Something to chat with Rob about maybe :)